modules/ROOT/pages/server_release_notes.adoc

```diff
@@ -45,6 +46,20 @@
 * xref:changes-in-8-0[Changes in 8.0]
 * xref:changes-in-7-0[Changes in 7.0]
 
+== Changes in 10.13.4
+
+Dear ownCloud administrator, find below the changes and known issues in ownCloud Server 10.13.4 that need your attention. You can also read the {oc-changelog-url}[full ownCloud Server changelog] for further details on what has changed.
+
+=== Show Alert about IoC Scanner to all Customers on Upgrade
+
+We now display IoC scanner instructions to all Customers (a valid license key needs to be present) during upgrade (console as well as Web Updater) and in the admin settings. Additionally, a new config.php option `indicators-of-compromise-scanner.confirmation` has been introduced which gives admins the possibility to suppress the instrcutions once the script has been correctly run on ALL ownCloud servers. https://github.com/owncloud/core/pull/41137[#41137]
+
+Background: the Indicators of Compromise (IoC) tool plays a vital role in identifying potential security threats or breaches. The tool analyzes your ownCloud 10 deployments and determines whether they have been compromised via the known vulnerabilities. It collects information from the Apache logs and identifies the signatures of potential exploits. As mentioned above, the tool has to be run ALL ownCloud servers in case of a clustered setup.
+
+=== 2FA Check on Controllers which are Annotated as @PublicPage and also Authenticated
```

Comment:
annotated and authenticated lower case? .... leaving this to tech writers .....

Reply:
No, we use headline capitalization according to the Chicago Manual of Style: https://headlinecapitalization.com/